config/routes.rb

```diff
@@ -4,12 +4,15 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   # Defines the root path route ("/")
   # root "articles#index"
+  mount_devise_token_auth_for 'User', at: 'api/v1/auth', controllers: {
```

Comment:
Why is this being changed now? Just asking

Reply:
Because having this code inside those namespaces causes devise routes to be renamed with the api_v1 prefix, which causes [this method (authenticate_user)](https://github.com/LeandroSoria2/TargetApi/blob/d45515c55024b5ef8daae56fcda495d519d8c9dd/app/controllers/api/v1/api_controller.rb#L5) (which was added because authentication was not working properly in the topics index) to have a different name & not be recognized. Thus, this is how it's structured in the [rails_api_base](https://github.com/rootstrap/rails_api_base/blob/master/config/routes.rb)